corehq/apps/app_manager/models.py

```diff
@@ -5152,6 +5152,12 @@ def import_app(app_id_or_source, domain, source_properties=None, validate_source
         del source['build_spec']
     app = cls.from_source(source, domain)
     app.cloudcare_enabled = domain_has_privilege(domain, privileges.CLOUDCARE)
+
+    for module in app.get_modules():
+        if isinstance(module, ReportModule):
+            for report_config in module.report_configs:
+                report_config.uuid = 'copy-' + report_config.uuid
```

Comment:
Fair enough*  Will merge once tests pass.

*although, did you know, we have code that's written to help with this problem: https://github.com/dimagi/dimagi-utils/blob/master/dimagi/utils/next_available_name.py ...but you'd still need to write code to gather all of the existing reports that include the uuid, so it's not quite trivial, understand if it isn't worth the effort